public/apps/configuration/panels/role-view/role-view.tsx

```diff
@@ -22,33 +22,152 @@ import {
   EuiSpacer,
   EuiTabbedContent,
   EuiTitle,
+  EuiPageContent,
+  EuiText,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPageBody,
+  EuiInMemoryTable,
+  EuiEmptyPrompt,
 } from '@elastic/eui';
 import { BreadcrumbsPageDependencies } from '../../../types';
 import { buildHashUrl } from '../../utils/url-builder';
 import { ResourceType, Action } from '../../types';
-
-const tabs = [
-  {
-    id: 'permissions',
-    name: 'Permissions',
-    disabled: false,
-    content: <Fragment>Permission working in progress</Fragment>,
-  },
-  {
-    id: 'users',
-    name: 'Users',
-    disabled: false,
-    content: <Fragment>Users working in progress</Fragment>,
-  },
-];
+import { getRoleMappingDeta, MappedUsersListing } from '../../utils/role-mapping-utils';
 
 interface RoleViewProps extends BreadcrumbsPageDependencies {
   roleName: string;
 }
 
+function getmappedUserColumns() {
```

Comment:
Minor: camel case for function name.

Reply:
Used constant named "mappedUserColumns" instead of function. Followed camel case